recipes/toolchain/deactivate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
So, one example is I have seen people install `gcc` and then `pip` install things. This can work sometimes. In other cases, it causes problems.

In the same way, I could see someone installing this with `conda`. I would rather we aim to have this behave in the least surprising way possible in that scenario than get complaints of weird bugs.